test/util/cnxml.js

```diff
@@ -3,7 +3,7 @@ export default global.cnxml = function cnxml(fragments) {
         throw new Error('cnxml`` does not support interpolation')
     }
 
-    return `<document xmlns="http://cnx.rice.edu/cnxml" cnxml-version="0.7" id="test" module-id="test">
+    return `<document xmlns="http://cnx.rice.edu/cnxml" cnxml-version="0.7" id="test" module-id="test" lang="en">
```

Comment:
Why is this change made here?

Reply:
I've forgotten to remove this. I was trying to figure out why tests are failing